docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md

```diff
@@ -67,6 +74,8 @@ process:
 ```
 :::
 
+Click on the **Download YAML File** button to get the yaml file.<br/><img src='img/integrations/hosts-operating-systems/Windows-YAML.png' alt="Windows-YAML" width="600"/>
```

Comment:
The image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reply:
fixed!